editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/TrialAccountForm.tsx

```diff
@@ -0,0 +1,91 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import { useFormik } from "formik";
+import { useToast } from "hooks/useToast";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+import Permission from "ui/editor/Permission";
+import SettingsDescription from "ui/editor/SettingsDescription";
+import SettingsSection from "ui/editor/SettingsSection";
+import { Switch } from "ui/shared/Switch";
+import { boolean, object, SchemaOf } from "yup";
+
+import { FormProps } from ".";
+
+interface TrialAccountForm {
+  isTrial: boolean;
+}
+
+export default function TrialAccountForm({
+  formikConfig,
+  onSuccess,
+}: FormProps) {
+  const formSchema: SchemaOf<TrialAccountForm> = object({
+    isTrial: boolean().required(),
+  });
+
+  const toast = useToast();
+
+
+  const formik = useFormik({
+    ...formikConfig,
+    validationSchema: formSchema,
+    onSubmit: async (values, { resetForm }) => {
+      try {
+        const isSuccess = await useStore.getState().updateTeamSettings({
+          isTrial: values.isTrial,
+        });
+
+        if (!isSuccess) throw Error("updateTeamSettings() failed");
+
+        onSuccess();
+        resetForm({ values });
+      } catch (error) {
+        console.error(error)
+        toast.error("Failed to update team's trial account status");
+      }
+    },
+  });
+
+  return (
+    <Permission.IsPlatformAdmin>
+      <Box component="form" onSubmit={formik.handleSubmit} mt={2}>
+        <SettingsSection background>
+          <Switch
+            label="Trial account"
+            name="isTrial"
+            variant="editorPage"
+            capitalize
+            checked={formik.values.isTrial}
+            onChange={() =>
+              formik.setFieldValue("isTrial", !formik.values.isTrial)
+            }
+          />
+          <SettingsDescription>
+            <p>Toggle this team between "trial" access and full access</p>
+            <p>
+              A trial team had limited access to PlanX features. They are not
+              able to publish services and turn them online.
```

Comment:
nit: "They are not able to turn services online" only ! Following on from standup chat about this, ability to publish regardless of "trial" also feels really important in keeping any templates up to date!

Reply:
Good catch! Once this is all completed (pretty much there I think) I'll certainly open a Slack thread seeking some ideas on the copy - "trial mode", "trial access" and "trial account" are all interchangeable used currently.